variables.tf

```diff
@@ -17,40 +17,35 @@ variable "infrastructure-support" {
   description = "The team responsible for managing the infrastructure. Should be of the form <team-name> (<team-email>)"
 }
 
-variable "ec_engine" {
-  description = "The engine which your ElastiCache Cluster will use. You have a choice of either 'redis' or 'memcached'. The default is 'redis'"
-  type        = "string"
-  default     = "redis"
-}
-
 variable "engine_version" {
   description = "The engine version that your ElastiCache Cluster will use. This will differ between the use of 'redis' or 'memcached'. The default is '4.0.10' with redis being the assumed engine."
   type        = "string"
   default     = "4.0.10"
 }
 
+variable "replication_group_description" {
+  description = "A short description of the replication group"
```

Comment:
I think we should instead set this automatically to something based on the provided tags.